src/www/ui/ajax-showjobs.php

```diff
@@ -267,22 +262,15 @@ protected function show($jobData, $page, $allusers)
           $allusers = GetParm("allusers",PARM_INTEGER);
           if ($allusers > 0){
             $statementName = __METHOD__."UploadRec";
-            $uploadRec = $dbManager->getSingleRow("select * from upload where upload_pk=$1",
+            $uploadRec = $this->dbManager->getSingleRow("select user_fk from upload where upload_pk=$1",
                 array($job['job']['job_upload_fk']),
                 $statementName);
 
             if (!empty($uploadRec['user_fk'])){
-              $statementName = __METHOD__."UserRec";
-              $userRec = $dbManager->getSingleRow("select * from users where user_pk=$1",
-                  array($uploadRec['user_fk']),
-                  $statementName);
+              $userName = "&nbsp;&nbsp;&nbsp;(" . htmlentities($this->userDao->getUserName($uploadRec['user_fk']), ENT_QUOTES) . ")";
             }else{
-              $statementName = __METHOD__."UserRec1";
-              $userRec = $dbManager->getSingleRow("select * from users where user_pk=$1",
-                  array($job['job']['job_user_fk']),
-                  $statementName);
+              $userName = "&nbsp;&nbsp;&nbsp;(" . htmlentities($this->userDao->getUserName($job['job']['job_user_fk']), ENT_QUOTES) . ")";
```

Comment:
Yes, I agree with you. I will add these changes for @shaheemazmalmmd, since the branch with his cherry-picked commits is on my repo.